snapshots/CaliburIsValidSignatureTest.json

```diff
@@ -1,8 +1,11 @@
 {
-  "isValidSignature_P256": "45287",
-  "isValidSignature_P256_typedData_notNested_safeERC1271Caller": "32773",
-  "isValidSignature_P256_withHook": "51296",
-  "isValidSignature_WebAuthnP256": "51973",
-  "isValidSignature_rootKey": "20822",
-  "isValidSignature_rootKey_typedData_notNested_safeERC1271Caller": "8304"
+  "isValidSignature_P256_personalSign": "35124",
+  "isValidSignature_P256_typedDataSign": "42515",
+  "isValidSignature_P256_typedData_notTypedDataSign_safeERC1271Caller": "295984",
+  "isValidSignature_P256_withHook": "48523",
+  "isValidSignature_WebAuthnP256_typedDataSign": "49106",
+  "isValidSignature_rootKey_personalSign": "10391",
+  "isValidSignature_rootKey_typedDataSign": "17780",
+  "isValidSignature_rootKey_typedData_notTypedDataSign_safeERC1271Caller": "16669",
+  "isValidSignature_sep256k1_typedDataSign": "38635"
```

Comment:
why is this so much more than the root key signing over typed data sign?

Reply:
both are k1 keys.. they should be relatively similar?

Reply:
ah I guess the root key we don't need to load anything from storage

Reply:
Yea that would be my assumption too